docs/index.md

```diff
@@ -114,7 +114,29 @@ If the issuer has more than one certificate for the same public key (e.g. partic
 
 Issuers SHALL publish their public keys as JSON Web Key Sets (see [RFC7517](https://tools.ietf.org/html/rfc7517#section-5)), available at `<<iss value from JWS>>` + `/.well-known/jwks.json`, with [Cross-Origin Resource Sharing (CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin) enabled.
 
-The URL at `<<iss value from JWS>>` SHALL use the `https` scheme and SHALL NOT include a trailing `/`. For example, `https://smarthealth.cards/examples/issuer` is a valid `iss` value (`https://smarthealth.cards/examples/issuer/` is **not**).
+The URL at `<<iss value from JWS>>` SHALL use the `https` scheme, and SHALL use the following TLS version 1.2 or 1.3 configurations:
+* For TLS 1.2, use a ciphersuite conforming to the following algorithm choices and _minimum_ cryptographic strength:
```

Comment:
```suggestion
* For TLS 1.2, support at least one ciphersuite conforming to the following algorithm choices and _minimum_ cryptographic strength:
```

Reply:
Meaning we're ok if the endpoint supports less secure options, allowing non-conforming clients to still get the keys?

Reply:
I think that's consistent with our internal discussion, yes -- here, I mainly intended it as a disambiguation of the way I already read "use a ciphersuite conforming".

Let me know if you think this is the wrong call.

Reply:
Given this is a new spec leading to (presumably) new implementations, enforcing a secure min bar makes sense to me. Should be ok for servers, even if keys are hosted by legacy systems. Clients (wallets and newly created apps) should also support this min bar. Bottom line: which legacy components that can't meet these recommendations are we trying to support?

Reply:
I think the potential issue is with existing web servers that support TLS 1.1 *in addition to* supporting TLS 1.2 with ciphersuites we recommend. The question is whether that's considered OK or not. Agree it's worth discussing on a call (I expect @isaacvetter may want to join).

Reply:
Superseded by ref to BCP 195 (TLS 1.1 is now deprecated)